osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -155,13 +155,20 @@ private void createRepeatPoints()
             var repeatPointDistance = Math.Min(Distance, length);
             var repeatDuration = length / Velocity;
 
+            bool sliderStart = true;

```

Comment:
Since this is local, I don't see it being set anywhere else. If that's the case is it required?